cli/client_test.go

```diff
@@ -38,7 +40,7 @@ func (tw *testWriter) Write(b []byte) (int, error) {
 // in AppServiceClient and DataServiceClient. It also returns testWriters that capture Stdout and
 // Stdin.
 func setup(asc apppb.AppServiceClient, dataClient datapb.DataServiceClient,
-	defaultFlags *map[string]string,
+	defaultFlags *map[string]string, authMethod string,
```

Comment:
I added functionality here to choose the authentication method when setting up tests. The majority will use "token" and can use "apiKey" if needed in specific tests (like the one I added).